src/barchart/BarChart.jsx

```diff
@@ -57,9 +55,19 @@ module.exports = React.createClass({
         .rangeRoundBands([0, props.width - sideMargins], 0.1);
 
     var trans = `translate(${ margins.left },${ margins.top })`;
-    
+
     return (
-      <Chart width={props.width} height={props.height} title={props.title}>
+      <Chart
+        viewBox={props.viewBox}
+        legend={props.legend}
+        data={props.data}
+        margins={props.margins}
+        colors={props.colors}
+        colorAccessor={props.colorAccessor}
```

Comment:
I think I just forgot to add it to `PieChart`, thanks for pointing that out. I'll go through all of them tomorrow morning and make sure they all have it where missing and send our a pr with those changes.